config/kubernetes-incubator/org.yaml

```diff
@@ -450,6 +451,7 @@ teams:
     - loburm
     - ncdc
     - piosz
+    - serathius
```

Comment:
If you don't mind, please add this part as a separate commit (can still be a part of this PR, just a separate commit). The big thing we need as individual commits is being added/removed from an org. 😬

Reply:
Sure.